package no.fintlabs.testrunner.exception

class InvalidAuthenticationException(clientName: String) : Throwable() {

}
